core/include/misc/fs_utils.h

```diff
@@ -0,0 +1,65 @@
+/**
+ * @file   fs_utils.h
+ *
+ * @section LICENSE
+ *
+ * The MIT License
+ *
+ * @copyright Copyright (c) 2017 Omics Data Automation Inc.
+ *                      (c) 2017 UCLA All rights reserved
```

Comment:
Thanks for the submission - this will be very helpful for us.
I still have to do a detailed review; however, I cannot accept the wording of this license (I'm not a lawyer, I just follow what our legal team specified).